aerosonicdb/__init__.py

```diff
@@ -1 +0,0 @@
-
```

Comment:
results of pre-commit run on this file and below, removes empty lines at end of file, empty characters at end of lines etc..